nextbike/datapreparation/data_clean.py

```diff
@@ -22,8 +19,8 @@ def data_cleaning(p_df_original):
     # Drop Duplicates and creating new df with only unique files
     p_df_original.drop_duplicates(subset=p_df_original.columns.difference(["p_lng", "p_lat"]), inplace=True)
     # Drop trip first/last
-    df_clean_unique_trip = p_df_original[(p_df_original["trip"] == "start") | (p_df_original["trip"] == "end")]
-    df_clean_unique_trip.sort_values(["b_number", "datetime"], inplace=True)
+    df_clean_unique_trip_unsorted = p_df_original[(p_df_original["trip"] == "start") | (p_df_original["trip"] == "end")]
+    df_clean_unique_trip = df_clean_unique_trip_unsorted.sort_values(["b_number", "datetime"])
```

Comment:
I think we should do it consistently... 
Therefore, replace all inplace=True by override of df or creation of ne df.

Reply:
replaced all inplace options in last commit :)